src/Data/List/Relation/Binary/Lex/NonStrict.agda

```diff
@@ -23,10 +23,12 @@ open import Relation.Nullary
 open import Relation.Binary
 import Relation.Binary.Construct.NonStrictToStrict as Conv
 
+import Data.List.Relation.Binary.Lex as Core
+
 ------------------------------------------------------------------------
 -- Publically re-export definitions from Core
 
-open import Data.List.Relation.Binary.Lex.Core as Core public
+open import Data.List.Relation.Binary.Lex.Core public
```

Comment:
Should this either be `open import Data.List.Relation.Binary.Lex.**NonStrict**.Core public` or `open Core public`?

Reply:
As it turns out, `Lex` defines both strict & non-strict lexicographic ordering.
The adjective in the module paths `Lex.Strict` and `Lex.NonStrict` refers to
whether the *lifted* order is one or the other. Basically only the `Strict` one
is of interest: the nonstrict one is using the non-strict-to-strict construction
to reuse the other.

This is probably worth revamping in the future.